podman/system/__init__.py

```diff
@@ -29,7 +29,10 @@ def get_info(api):
 
 
 def show_disk_usage(api):
-    """Return information about disk usage for containers, images and volumes"""
+    """
+    Return information about disk usage for containers,
+    images and volumes
+    """
```

Comment:
I think you updated this because of line length? Since we're using pylint, it should be 100. I'd leave this.

Reply:
You're right. I define 79 characters as maximum length of the line, in the other hand Black defaults to 88 characters per line.